templates/zerver/help/index.md

```diff
@@ -91,7 +89,7 @@ as an **organization**.
 * [Rename a stream](/help/rename-a-stream)
 * Preview a stream (not implemented)
 * [Unsubscribe from a stream](/help/unsubscribe-from-a-stream)
-* [Change who can join a stream](/help/change-who-can-join-a-stream)
+* [Change the accessibility of a stream](/help/change-the-accessibility-of-a-stream)
```

Comment:
I still think accessibility in terms of web design means "designed with disabled people in mind", e.g.

https://en.wikipedia.org/wiki/Web_accessibility
https://www.w3.org/standards/webdesign/accessibility
https://www.apple.com/lae/accessibility/